ch2.txt

```diff
@@ -30,6 +23,11 @@ CHAPTER 2. WAGING WAR
 
 16. Now in order to kill the enemy, our men must be roused to anger; that there may be advantage from defeating the enemy, they must have their rewards. 
 
+asdf
+asdf
+asdf
+adsf
+
```

Comment:
This is another spanned inline comment attached to the newly added text, that GitHub will interpret as a code suggestion [0], suggesting the added text.

```suggestion
arst
arst
arst
arst
```

[0]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request